spec/models/vmdb_database_setting_spec.rb

```diff
@@ -67,8 +62,8 @@
     :vmdb_database_id
   ].each do |field|
     it "has a #{field}" do
-      setting = VmdbDatabaseSetting.all.first
-      expect(setting.send(field)).to be_truthy
+      setting = VmdbDatabaseSetting.first
+      expect(setting).to respond_to(field)
```

Comment:
This is how it was written for the same code on [vmdb_database_connection_spec:120](https://github.com/ManageIQ/manageiq/blob/master/spec/models/vmdb_database_connection_spec.rb#L120)